### DISCUSSION TITLE
Correct spelling error on commit message?

### DISCUSSION THREAD

**Discussion by JStateson** [community/community#21711] [New to GitHub]
<p>I committed but had a really bad spelling error.  I did not see any way to correct it at the remote (here) so I “touched” the module  and did another commit after correcting the spelling.  Is there a way to do this without going through another commit?</p>
<p>Thanks for looking!</p>

**Comment by seveas** [comment, marked as answer]
<p>You can use</p>
<pre><code>git commit --amend
</code></pre>
<p>to modify the latest commit. If you had already pushed the ‘bad’ commit, you would then need to force-push it, which may or may not be what you want.</p>
